modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/Wrapper.js

```diff
@@ -39,24 +40,25 @@ function Wrapper() {
 
 			<CartViews.OrderButton />
 
-			{requestQuoteEnabled && !!cartItems.length && (
-				<div className="request-quote-wrapper">
-					<ClayButton
-						block={true}
-						className="btn-md request-quote"
-						displayType="secondary"
-						onClick={() => {
-							return liferayNavigate(orderDetailURL);
-						}}
-					>
-						<span className="text-truncate-inline">
-							<span className="text-truncate">
-								{Liferay.Language.get('request-a-quote')}
+			{(requestQuoteEnabled || hasPriceOnApplication(cartItems)) &&
+				!!cartItems.length && (
+					<div className="request-quote-wrapper">
+						<ClayButton
+							block={true}
+							className="btn-md request-quote"
+							displayType="secondary"
+							onClick={() => {
+								return liferayNavigate(orderDetailURL);
+							}}
```

Comment:
Move this to a new request quote button component

Reply:
Done